aiidalab_widgets_base/viewers.py

```diff
@@ -156,6 +163,61 @@ def __init__(self, parameter, downloadable=True, **kwargs):
         super().__init__([self.widget], **kwargs)
 
 
+class Representation(ipw.VBox):
```

Comment:
Make a more concrete name. It is too generic to call it `Representation`.

Reply:
done